examples/c5g7_transient/rod_translation/mgxs_lib.py

```diff
@@ -0,0 +1,282 @@
+#coding=utf-8
```

Comment:
File is almost identical to `density_change/mgxs_lib.py`. Consider removing and adding a symbolic link instead